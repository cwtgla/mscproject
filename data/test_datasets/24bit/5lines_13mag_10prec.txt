8191.1313813
2000.2008
0.0024
-245.42424
-8191.249294