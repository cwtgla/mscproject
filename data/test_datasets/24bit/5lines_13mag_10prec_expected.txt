127
252
131
31
64
200
0
0
2
131
213
168
255
252
249